src/lib/components/manualOnOffCluster.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+    import { changePlugState, getPlugState } from '$lib/backend-api';
+    import type { PowerState } from '$lib/response-types';
+    import { onMount } from 'svelte';
+    import { Lightbulb, LightbulbOff } from '@lucide/svelte';
+
+    export let background_styles: string | null = null;
+    export let indicator: boolean = true;
+
+    let loading = true;
+    let currentPowerState: boolean = false;
```

Comment:
Hier ist die Svelte 5 Syntax für props/state bevorzugt. Auch wenn ich die nicht immer verwendet habe :sweat_smile:  (also muss auch nicht unbedingt)

```suggestion    
    let {
        background_styles = null,
        indicator = true,
    }: { background_styles?: string | null; indicator?: boolean } = $props();

    let loading = $state(true);
    let currentPowerState: boolean = $state(false);
```

Reply:
Ja können wir gerne machen :+1:, hatte mich tbh noch nicht wirklich mit Svelte 5 und entsprechend nicht mit Runes beschäftigt.